bin/epaa.py

```diff
@@ -642,7 +642,7 @@ def get_protein_ids_for_transcripts(idtype, transcripts, ensembl_url, reference)
     else:
         input_lists += [transcripts]
 
-    attribut_swissprot = "uniprot_swissprot_accession" if reference == 'GRCh37' else 'uniprot_swissprot'
+    attribut_swissprot = "uniprot_swissprot_accession" if reference == 'GRCh37' else 'uniprotswissprot'
```

Comment:
We changed to GRCh38 right? Is this then still right?

Reply:
> We changed to GRCh38 right? Is this then still right?

I am not sure if I fully understand this. :) I changed the "swissprot database field name" for `GRCh38` to `uniprotswissprot` since this is the one used in the new archive version we are using now.